### DISCUSSION TITLE
PrismJS

### DISCUSSION THREAD

**Discussion by Liumingxun** [catppuccin/catppuccin#2374] [Port Requests]
### Will you make the port?

- [ ] Yes, I'm willing to make the port

### How can the suggested port be themed?

It may be similar to vs code syntax highlighter?

### Any additional comments?

[the website link here](https://prismjs.com/)
[the other themes here](https://github.com/PrismJS/prism-themes)

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by uncenter** [comment]
Not to say we shouldn't create a theme but if you are looking for a good syntax highlighter https://shiki.style/ is the best there is!

**Reply by Liumingxun** [reply]
Wow, thanks for it, I will be try it!

**Reply by uncenter** [reply]
No worries! And of course Shiki has all the Catppuccin themes built in :)

**Reply by uncenter** [reply]
FYI I did eventually make a port for it if you are ever needing a use for it: https://github.com/uncenter/ctp-prismjs
